### ISSUE TITLE
About datasets

### ISSUE THREAD

**Issue by SuFame920** [NUSTM/SemEval-2024_ECAC#4]
Hello author! Thank you very much for your team's outstanding contribution to labeling the ECF 2.0 dataset! However, I have some questions I would like to ask you: 1. Is ECF2.0 a subset of MELD? 2. The readme mentions that the training set for this task is the whole ECF1.0, so this means that ECF did not label the whole MELD right?

**Comment by ffcarina** [comment]
@SuFame920 
Thanks for your interest in our work!
1. ECF 1.0 is indeed based on the entire MELD dataset. The difference in the statistics is because **we pre-processed and filtered the original MELD to improve its quality and consistency**. This process included, but was not limited to: correcting errors in the transcriptions; splitting conversations that spanned across different scenes; removing conversations where there was an inconsistency across the transcriptions and timestamps. 
2. ECF 2.0 is an extension of ECF 1.0. The relationship is: **ECF 2.0 = ECF 1.0 + an extended test set**. This extended test set contains new dialogues that we collected ourselves from the original Friends TV series. It was created primarily to be used as the official evaluation data for the SemEval-2024 Task 3.

**Comment by SuFame920** [comment]
Gotta it! Thanks !
